src/operators/implementation/onnx/operator__onnx__elu__6__T_tensor_float.c

```diff
@@ -1,10 +1,58 @@
-//this file was generated by ../../../../../scripts/onnx_generator/OperatorStubs.py
-#include "operators/onnx/operator__onnx__elu__6.h"
-
-operator_status
-operator__onnx__elu__6__T_tensor_float(
-    node_context *ctx
-)
-{
-    return OP_ENOSYS;
-}
\ No newline at end of file
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <math.h>
+#include "trace.h"
+#include "utils.h"
+
+ operator_status operator__onnx__elu__6__T_tensor_float(
+     node_context *ctx
+ )
+ {
+
+   Onnx__TensorProto *X = searchInputByName(ctx, 0);
+
+   Onnx__TensorProto *Y = searchOutputByName(ctx, 0);
+
+   TRACE_LEVEL0("Calling operator_elu\n");
+
+   if (0){
+     /* TODO: Check some conditions. For example if a specific
+      * functionality is not supported */
+     return -1;
+   }
+
+   debug_print_dims(X->n_dims, X->dims);
+
+   /* TODO Alpha is not always float */
+   /* Default value */
+   float alpha = 1.0;
+
+   /* If there is an attribute, its alpha */
+   if (ctx->onnx_node->n_attribute == 1){
+     alpha = ctx->onnx_node->attribute[0]->f;
+   }
```

Comment:
could you use the `searchAttributeNyName` helper here? afaik specifying unneeded attributes is valid